kythe/typescript/kythe.ts

```diff
@@ -0,0 +1,211 @@
+/**
+ * @fileoverview TypeScript typings of the Kythe schema.
+ * @author Kythe Authors
```

Comment:
I think this is can be omitted.

Now that I think of it, I think you need to copy the license block into this file too.
So like

/*
 * Copyright [paste from other file ]
 */

/** @fileoverview TypeScript definitions of types found in the Kythe schema. */

Reply:
Done.